packages/remix-cloudflare-workers/index.ts

```diff
@@ -1,12 +1,6 @@
-import { installGlobals } from "./globals";
-
-export { createCloudflareKVSessionStorage } from "./sessions/cloudflareKVSessionStorage";
```

Comment:
Although I agree this doesn't belong here, this will probably be a breaking change for anyone running remix on cloudflare. For now, we should probably just deprecate importing `createCloudflareKVSessionStorage` from here and tell them to get it from `@remix-run/cloudflare` instead.

Reply:
`import { createCloudflareKVSessionStorage } from "remix"` should still work for existing apps since we still have `createCloudflareKVSessionStorage` in the magic exports (until we decide to do a breaking change to remove the magic exports).

It would only break if users did `import { createCloudflareKVSessionStorage } from "@remix-run/cloudflare-workers"`, but I think we agreed that changing the API `@remix-run/cloudflare-workers` isn't considered a breaking change since no one should be importing directly from `@remix-run/cloudflare-workers`.

Reply:
Ah yes, of course. You're right. Thanks for the reminder. I was just reminding you of this yesterday, haha ;)